wiki/Rules/Visual_Content_Considerations/zh.md

```diff
@@ -0,0 +1,113 @@
+# 视觉内容注意事项
+
+根据[社区规定第 5 条](/wiki/Rules#community-rules)，上传的谱面中的所有视觉内容必须适合全年龄游戏。这包含谱面背景图，皮肤元素，故事板素材等等。
+
+为了保持简介，社区规定第 5 条被解释为不允许 NSFW/18+ 内容以及「疑问性」的内容。
+
+我们请你在决定一件事物是否适合 osu! 的时候考虑以下内容，并同时记住我们是一个面向多元文化用户群的全年龄游戏，他们对媒体内容的期望会具有很大差别。
+
+## 重要事项
+
+作为基准，osu! 意图遵守[泛欧洲游戏信息组织（PEGI）12 分级](https://pegi.info/what-do-the-labels-mean)并带有一定追加，在以下解释。当两者冲突时，本文中 osu! 自己的「村规」总是优先。
+
+所有实觉内容都应「凭空」被判断，即，它们应当根据图像中直接展示的内容被判断，像一个不知道其来源材料或其它谱面背景的人那样。
+
+先前被允许不代表在未来也会被允许，尤其是在旧谱面因为技术原因无法在后来修改视觉内容的环境下。
+
+## 不合适的视觉内容
+
+视觉内容（即：谱面背景图，皮肤，谱面包中发行的其它图片或者视频相关的素材）**不得**含有以下任何内容：
+
+### 严重暴力
+
+这包含对人类、拟人角色或动物的任何暴力。
+
+任何描述凝血、肢体残缺、致残、重度伤害的图像，或者图形描述具有明确死亡方式的尸体（如绞死、窒息、出血），都是不可接受或容忍的。
+
+### 性暗示
+
+这指对性行为的模仿，或者以下流的方式索求。动漫环境下的这些内容通常称为「ecchi」。
+
+任何模仿性行为或者涉及模仿生殖器的物体的图像都是不允许的。
+
+### 性明示
+
+任何高度暗示性行为或直接暗示迫在眉睫的性欲的图像都是不允许的。这包括「来啊」类型的邀请或以其他方式高度暗示即将进行性接触的图像。
+
+这也适用于「意外」姿态，例如描绘在洗澡被偷拍的人物的图像，或其他此类情况。
+
+### 未成年性行为
+
+如果人物的年龄不能由其外观特征立刻判断（例如，无第二性征强烈指示角色为未成年），任何能被合理联想至性行为的内容是完全禁止的。
+
+这包含以「loli」风格画出的有疑问性的人物，无论其设定年龄如何。
+
+以任何方式描写未成年性行为在世界上大部分地方都高度违法，因此其被严禁。
+
+### 色情内容或者性行为图像
+
+描写正在进行的性行为、前导动作和生殖器官可见的正面裸体都不允许，无一例外。
+
+这扩展到明确带有色情性质的服装，或者设计为在性目的下使用的服装，例如捆绑绳或对情趣内衣的完整描写。
+
+另外，这也扩展到可被合理评估为氛围以挑逗或性暗示为主的整个艺术作品。
+
+### 吸毒
+
+这包括描述使用或者存在吸毒用具，如用过的针头等任何广泛用于管制精神药物的工具。
+
+出于一致性，任何全球规定不一致的药品（比如大麻）也被视为「非法」。与这些药品及其使用相关的图像也不被允许。
+
+为了这一规定的清晰，对于在大部分国家并没有被禁止或管制的药物，明确指示或描绘其滥用（如糖浆 / 瘦肉精 / 丙胺类滥用）也被视为「非法」。
+
+饮酒与吸烟在合理合量的情况下可以少量描述。
+
+简单地说，大多数 / 所有关于成瘾性药物使用的提及通常都不行，应该避免。
+
+### 蓄意煽动性内容
+
+这包括使用旨在蓄意煽动的文化或宗教符号，比如纳粹或法西斯符号（万字符），反犹太符号，或者其它鼓励伤害和歧视其它社会、种族、文化或政治群体的图标、符号或语句。
+
+## 合适的视觉内容
+
+在 osu! 中使用的理想内容应是和善的，适合被全年龄的观众观看。如果有疑问，查询当地的分级规定（如果可用）————如果其被视为适合 12 岁以上的观众（PEGI 12 或美国及英联邦国家的「PG」分级），应当可被广泛接受。
+
+视觉内容可以在符合常识和诚意下包含**一定量的**以下内容：
+
+### 泳装
+
+描绘*合理的*泳装是允许的，但不能太过分或违反上述的性暗示规定。
+
+过分是一个模糊的词汇，但其通常指具有充分争议性的内容，比如角色穿着小到不能覆盖其身体的泳装，或者说体表在场景中占比过大。
+
+过小或者过度暴露的泳装如果不是出于诚意则一般不被允许，其被认为对相当数量的用户不合适。
+
+### 轻度到中度的暴力
+
+涉及到战役、打斗或其它可能引起轻伤的动作的场景可以接受，如果其中没有大量的凝血、流血或者使人不适的残肢。
+
+### 艺术性裸体
+
+任何以纯艺术目的创作的，在非性环境下的，描绘部分裸体的传统或人体艺术作品，经过 Global Moderation Team 的审查后可能会被允许。
+
+然而，艺术性裸体必须与上下文有一定的相关性，才能适用于此条例外。
+
+一个合理应用的例子是来自 Pendulum 的*沉浸式*专辑封面，其在画面中央描绘了裸体男女将手按在一起的倾斜视角。该图标相对较小、典雅、可辨认，由于其非性相关的本质且与该专辑中的歌曲相关，被视为允许。
+
+### 浪漫、调情或亲密的感情
+
+以叙事上下文描绘角色间的浪漫或正常的亲密行为可被接受，如果其没有违反以上规定。
+
+这也包含玩性而不是过激的「轻浮」行为，且不得违反以上规定。
+
+## 更多
+
```

Comment:
It looks like you are missing the contents of this section (see lines 105 - 107 in the English version).